### PR TITLE
Add missing platform_family attribute

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Review by IanMadd — APPROVED** [review]

File: themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md [hunk 1]
```diff
@@ -21,38 +21,42 @@ Infra Client run. The most commonly accessed automatic attributes are:
 <td>The platform on which a node is running. This attribute helps determine which providers will be used.</td>
 </tr>
 <tr class="even">
+<td><code>node['platform_family']</code></td>
+<td>The platform family is a Chef Infra specific grouping of similar platforms where cookbook code can often be shared. For example `rhel` includes Red Hat Linux, Oracle Linux, CentOS, and several other platforms that are nearly identical to Red Hat Linux.</td>
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md:25]:
```suggestion
<td>`platform_family` is a Chef Infra attribute that groups together similar platforms in order to support sharing cookbook code. For example `[platform_family][rhel]` includes Red Hat Linux, Oracle Linux, CentOS, and several other platforms that are nearly identical to Red Hat Linux.</td>
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md (hunk 1 above)
Comment (tas50) [reply themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md:25]: InSpec doesn't actually use this value. In a perfect world that would be true, but this is 100% an infra thing.

Comment (kagarmoe) [reply themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md:25]: OmO. It calls `platform_family` to match `os.family`.

Comment (tas50) [reply themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md:25]: os.family and platform_family do not match.  It's a mess, but we're stuck with it unless one of you has a time machine.

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**